ocaml-lsp-server/src/document.mli

```diff
@@ -74,4 +74,5 @@ type type_enclosing =
   ; doc : string option
   }
 
-val type_enclosing : t -> Msource.position -> type_enclosing option Fiber.t
+val type_enclosing :
+  ?verbosity:int -> t -> Msource.position -> type_enclosing option Fiber.t
```

Comment:
can you make this argument mandatory? i don't find optional arguments as useful most of the time.